### PR TITLE
Updated the download link admx files Windows 10

### PR DESCRIPTION
**Pull request by joinimran**
Added link for April 2018 and Oct 2018 ADMX files.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2223

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: browsers/internet-explorer/ie11-deploy-guide/administrative-templates-and-ie11.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ Regardless which tool you're using to edit your Group Policy settings, you'll ne
 -   **If you're using GPMC with Advanced Group Policy Management (AGPM).** See [Checklist: Create, Edit, and Deploy a GPO](https://go.microsoft.com/fwlink/p/?LinkId=214522) for step-by-step instructions about how to check out a GPO from the AGPM archive, edit it, and request deployment.
 
 ## Related topics
-- [Administrative templates (.admx) for Windows 10 download](https://go.microsoft.com/fwlink/p/?LinkId=746579)
+- [Administrative templates (.admx) for Windows 10 April 2018 Update](https://www.microsoft.com/en-us/download/details.aspx?id=56880)
+- [Administrative templates (.admx) for Windows 10 October 2018 Update](https://www.microsoft.com/en-us/download/details.aspx?id=57576
```
Comment (JohanFreelancer9) [inline browsers/internet-explorer/ie11-deploy-guide/administrative-templates-and-ie11.md]:
```suggestion
- [Administrative templates (.admx) for Windows 10 October 2018 Update](https://www.microsoft.com/en-us/download/details.aspx?id=57576)
```

File: browsers/internet-explorer/ie11-deploy-guide/administrative-templates-and-ie11.md [hunk 2]
```diff
@@ -77,6 +77,7 @@ Regardless which tool you're using to edit your Group Policy settings, you'll ne
...
+)
```
Comment (JohanFreelancer9) [inline browsers/internet-explorer/ie11-deploy-guide/administrative-templates-and-ie11.md]:
```suggestion
```

**Comment by joinimran** [comment]
Ready to go online.

**Comment by mypil** [comment]
@dansimp - Can you please assist to check, approve and merge the content of this PR based on issue #2223?

Thank you.

File: browsers/internet-explorer/ie11-deploy-guide/administrative-templates-and-ie11.md [hunk 3]
```diff
@@ -77,6 +77,6 @@ Regardless which tool you're using to edit your Group Policy settings, you'll ne
 -   **If you're using GPMC with Advanced Group Policy Management (AGPM).** See [Checklist: Create, Edit, and Deploy a GPO](https://go.microsoft.com/fwlink/p/?LinkId=214522) for step-by-step instructions about how to check out a GPO from the AGPM archive, edit it, and request deployment.
 
 ## Related topics
-- [Administrative templates (.admx) for Windows 10 download](https://go.microsoft.com/fwlink/p/?LinkId=746579)
+- [Administrative templates (.admx) for Windows 10 April 2018 Update](https://www.microsoft.com/en-us/download/details.aspx?id=56880)
```
Comment (nenonix) [inline browsers/internet-explorer/ie11-deploy-guide/administrative-templates-and-ie11.md]:
```suggestion
- [Administrative templates (.admx) for Windows 10 April 2018 Update](https://www.microsoft.com/download/details.aspx?id=56880)
```

File: browsers/internet-explorer/ie11-deploy-guide/administrative-templates-and-ie11.md [hunk 4]
```diff
@@ -77,6 +77,6 @@ Regardless which tool you're using to edit your Group Policy settings, you'll ne
...
+- [Administrative templates (.admx) for Windows 10 October 2018 Update](https://www.microsoft.com/en-us/download/details.aspx?id=57576)
```
Comment (nenonix) [inline browsers/internet-explorer/ie11-deploy-guide/administrative-templates-and-ie11.md]:
```suggestion
- [Administrative templates (.admx) for Windows 10 October 2018 Update](https://www.microsoft.com/download/details.aspx?id=57576)
```

**Comment by mypil** [comment]
@joinimran - please merge the commit on the copy/edit review made by  nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by joinimran** [comment]
Ready to go.

**Comment by mypil** [comment]
@Dansimp @AndreaBarr - Can you please assist to check, approve and merge the content of this PR based on issue #2223?

Thank you.

**Pull request merged**